docs/sources/get-started/quick-start.md

```diff
@@ -48,34 +45,22 @@ The Docker Compose configuration runs the following components, each in its own
 
 <!-- INTERACTIVE page intro.md END -->
 
+<!-- INTERACTIVE ignore START -->
 ## Before you begin
 
 {{< admonition type="tip" >}}
 Alternatively, you can try out this example in our interactive learning environment: [Loki Quickstart Sandbox](https://killercoda.com/grafana-labs/course/loki/loki-quickstart).
 
 It's a fully configured environment with all the dependencies already installed.
 
-![Interactive](https://raw.githubusercontent.com/grafana/killercoda/prod/assets/loki-ile.svg)
-{{< /admonition >}}
-
-- Install [Docker](https://docs.docker.com/install)
-- Install [Docker Compose](https://docs.docker.com/compose/install)
-
-## Interactive Learning Environment
-
-{{< admonition type="note" >}}
-The Interactive Learning Environment is in trial.
+![Interactive](/media/docs/loki/loki-ile.svg)
 
 Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
 {{< /admonition >}}
 
-Try out this demo within our interactive learning environment: [Loki Quickstart Sandbox](https://killercoda.com/grafana-labs/course/loki/loki-quickstart)
-
-- You must have a free Killercoda account to verify you aren't a bot.
-- Tutorial instructions are located on the left-side of the screen.
-  Click to move on to the next section.
-- All commands run inside the interactive terminal.
-- You can access Grafana with the URL links provided within the sandbox.
+- Install [Docker](https://docs.docker.com/install)
```

Comment:
I would move these two bullet points above the Tip admonition.  Right now the admonition reads "Alternatively, you can try out...." but I think it makes more sense for that to come **_after_** we tell users they have to install Docker and Docker Compose.

Reply:
Great catch! Should be updated now